package.json

```diff
@@ -48,7 +45,7 @@
     "react-dom": "^17.x || ^18.x"
   },
   "dependencies": {
-    "@prezly/sdk": "^7.11.0-0",
+    "@prezly/sdk": "^11.8.1-cjs",
```

Comment:
Also, we might get into trouble if we want to update the SDK version, since we'll need to release a separate CJS version each time. This will also affect themes, which also require SDK dependency.

It might be better to add secondary CJS build to the SDK package as a workaround for now. I was the one who proposed moving to ESM some time ago, but now I see that it was too early to pull the switch 😕

Reply:
What's the problem with using ESM here?

Reply:
I already did try moving everything to ESM, but consuming Next.js themes just refused to work with any combination of dependencies, if there was some ESM code in place. I think this has to do with the fact that we use SDK both directly and through theme kit, but even when aligning the versions, it still refused to work.

Reply:
:disappointed:

Reply:
@oleg-semyonov @e1himself i created a pr to add cjs config for sdk, could you take a look https://github.com/prezly/javascript-sdk/pull/183